data/linux-bridge/003-bridge-marker.yaml

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
Where is Namespace ?

Reply:
Do you mean [this](https://github.com/kubevirt/cluster-network-addons-operator/pull/458/files/61c4cb119f80b2890336cf0ab5933eb275ac00b4#diff-0570096c4288aecf7e8ceb569e77cf7eR6)?

Reply:
i think we have all the Namespaces